apps/studio/prisma/scripts/updateUserPermissions.ts

```diff
@@ -0,0 +1,25 @@
+import { db, RoleType } from "~/server/modules/database"
+
+export const updateUserPermissions = async (siteId: number, role: RoleType) => {
```

Comment:
IMO this function name is potentially misleading as it doesn't require us to pass in any `user.id` but rather what it's doing is update the user permissions of **ALL** users of the site

Reply:
sure, lemme update the name to `updateAllUserPermissionsForSite`